README.md

```diff
@@ -1 +1,7 @@
-# online-book-store
\ No newline at end of file
+# online-book-store
+
+There is a list of all entities that presented in the project:
```

Comment:
Are these changes really needed?

Reply:
Each task have new information and logic to implement, so I try to get information to describe my project.